### PR TITLE
[PPMiniLM P0] Add PretrainedConfig and unit tests

### PR DESCRIPTION
**Pull request by LiuChiachi**
### PR types

Others
### PR changes

APIs | Models|Others
### Description

Add PretrainedConfig and unit tests

- [x] PretrainedConfig
- [x] unit tests
- [x] examples/model_compression/pp-minilm 
- [x] Remove `add_faster_tokenizer_op` in PPMiniLM, because the usage of `FasterTokenizer` is updated now.

And format the code style of `model_zoo/tinybert/task_distill.py`

Run unit tests：
```shell
RUN_SLOW_TEST=true pytest ./tests/transformers/ppminilm

### CODE REVIEW

File: paddlenlp/transformers/ppminilm/modeling.py [hunk 1]
```diff
@@ -36,23 +37,14 @@ class PPMiniLMEmbeddings(nn.Layer):
...
-    ):
+    def __init__(self, config: PPMiniLMConfig):
         super(PPMiniLMEmbeddings, self).__init__()
 
-        self.word_embeddings = nn.Embedding(vocab_size, hidden_size, padding_idx=pad_token_id, weight_attr=weight_attr)
-        self.position_embeddings = nn.Embedding(max_position_embeddings, hidden_size, weight_attr=weight_attr)
-        self.token_type_embeddings = nn.Embedding(type_vocab_size, hidden_size, weight_attr=weight_attr)
-        self.layer_norm = nn.LayerNorm(hidden_size)
-        self.dropout = nn.Dropout(hidden_dropout_prob)
+        self.word_embeddings = nn.Embedding(config.vocab_size, config.hidden_size)
```
Comment (sijunhe) [inline paddlenlp/transformers/ppminilm/modeling.py]: padding_idx这个参数是不是漏了？

File: tests/transformers/ppminilm/test_modeling.py [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+            input_mask,
+            sequence_labels,
+            token_labels,
+            choice_labels,
+        ) = config_and_inputs
+        inputs_dict = {"input_ids": input_ids, "token_type_ids": token_type_ids, "attention_mask": input_mask}
+        return config, inputs_dict
+
+
+@parameterized_class(
```
Comment (sijunhe) [inline tests/transformers/ppminilm/test_modeling.py]: return_dict和labels当前模型都没有，可以删

File: tests/transformers/ppminilm/test_modeling.py [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+
+    def create_and_check_model(
+        self,
+        config: PPMiniLMConfig,
+        input_ids,
+        token_type_ids,
+        input_mask,
+        sequence_labels,
+        token_labels,
+        choice_labels,
```
Comment (sijunhe) [inline tests/transformers/ppminilm/test_modeling.py]: 当前模型没有做labels的话，所有的这些labels输入都可以删

File: tests/transformers/ppminilm/test_modeling.py [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+    ):
+        model = PPMiniLMForQuestionAnswering(config)
+        model.eval()
+        result = model(
+            input_ids,
+            # attention_mask=input_mask,
```
Comment (sijunhe) [inline tests/transformers/ppminilm/test_modeling.py]: attention_mask还是打开，确认一下能work

File: tests/transformers/ppminilm/test_tokenizer.py [hunk 5]
```diff
@@ -0,0 +1,347 @@
...
+)
+
+from ...testing_utils import slow
+from ...transformers.test_tokenizer_common import (
+    TokenizerTesterMixin,
+    filter_non_english,
+)
+
+
+class BertTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
```
Comment (sijunhe) [inline tests/transformers/ppminilm/test_tokenizer.py]: 这里还是BertTokenizer，需要改

File: paddlenlp/transformers/ppminilm/configuration.py [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+    },
+}
+
+PPMINILM_PRETRAINED_RESOURCE_FILES_MAP = {
+    "model_state": {
+        "ppminilm-6l-768h": "https://bj.bcebos.com/paddlenlp/models/transformers/ppminilm-6l-768h/ppminilm-6l-768h.pdparams",
+    },
+    "vocab_file": {
+        "ppminilm-6l-768h": "https://bj.bcebos.com/paddlenlp/models/transformers/ppminilm-6l-768h/vocab.txt",
+    },
```
Comment (sijunhe) [inline paddlenlp/transformers/ppminilm/configuration.py]: 确认一下这个vocab_file需不需要，别的模型都是统一放到tokenizer下面的

File: paddlenlp/transformers/ppminilm/modeling.py (hunk 1 above)
Comment (LiuChiachi) [reply paddlenlp/transformers/ppminilm/modeling.py]: 感谢，已补充，之前考虑到可以省略就去掉了，还是保持一致更好

File: tests/transformers/ppminilm/test_modeling.py (hunk 2 above)
Comment (LiuChiachi) [reply tests/transformers/ppminilm/test_modeling.py]: 感谢提醒，已经删掉

File: tests/transformers/ppminilm/test_modeling.py (hunk 4 above)
Comment (LiuChiachi) [reply tests/transformers/ppminilm/test_modeling.py]: 感谢提示，已经补充

File: tests/transformers/ppminilm/test_modeling.py (hunk 3 above)
Comment (LiuChiachi) [reply tests/transformers/ppminilm/test_modeling.py]: 感谢提醒，已经删掉

File: paddlenlp/transformers/ppminilm/configuration.py (hunk 6 above)
Comment (LiuChiachi) [reply paddlenlp/transformers/ppminilm/configuration.py]: 感谢提醒，已经删掉。这里的`vocab_file`现在可以删掉了。

File: tests/transformers/ppminilm/test_tokenizer.py (hunk 5 above)
Comment (LiuChiachi) [reply tests/transformers/ppminilm/test_tokenizer.py]: 已经修改为`PPMiniLMTokenizer`的单测

**Review by sijunhe — APPROVED** [review]
lgtm
